oak/server/dev/dev_oak_manager.cc

```diff
@@ -38,13 +38,43 @@ grpc::Status DevOakManager::CreateApplication(grpc::ServerContext* context,
   std::string application_id = NewApplicationId();
   LOG(INFO) << "Creating app with with id: " << application_id;
 
-  InitializeAssertionAuthorities();
-  runtime_->InitializeServer(
-      request->application_configuration(),
-      asylo::EnclaveServerCredentials(asylo::BidirectionalNullCredentialsOptions()));
+  auto runtime = absl::make_unique<OakRuntime>();
+  auto status = runtime->Initialize(request->application_configuration());
+  if (!status.ok()) {
+    return status.ToOtherStatus<grpc::Status>();
+  }
+
+  grpc::ServerBuilder builder;
+  int port;
+
+  // Uses ":0" notation so that server listens on a free port.
+  builder.AddListeningPort(
+      "[::]:0", asylo::EnclaveServerCredentials(asylo::BidirectionalNullCredentialsOptions()),
+      &port);
+  builder.RegisterService(runtime->GetGrpcService());
+
+  // Add a completion queue and a generic service, in order to proxy incoming RPCs to the Oak Node.
+  auto completion_queue = builder.AddCompletionQueue();
+
+  // Register an async service
+  auto module_service = absl::make_unique<grpc::AsyncGenericService>();
+  builder.RegisterAsyncGenericService(module_service.get());
+
+  auto server = builder.BuildAndStart();
+  if (!server) {
+    grpc::Status(grpc::INTERNAL, "Failed to start gRPC server");
+  }
+
+  // moves ownership of unique pointers
```

Comment:
```suggestion
  // Move ownership of unique pointers.
```

(but also the comment seems redundant anyways).

Reply:
Done